backend/import_status/get.py

```diff
@@ -57,13 +58,40 @@ def get_import_status(event: JsonObject) -> JsonObject:
         )
 
     response_body = {
-        "validation": {"status": step_function_resp["status"]},
+        "validation": {
+            "status": step_function_resp["status"],
+            "errors": get_step_function_validation_results(step_function_resp["input"]),
+        },
         "upload": upload_response,
     }
 
     return success_response(200, response_body)
 
 
+def get_step_function_validation_results(step_function_input: str) -> JsonList:
+    input_json = json.loads(step_function_input)
+    hash_key = f"DATASET#{input_json['dataset_id']}#VERSION#{input_json['version_id']}"
+
+    errors = []
+    for validation_item in ValidationResultsModel.validation_outcome_index.query(
+        hash_key=hash_key,
+        range_key_condition=ValidationResultsModel.result == ValidationResult.FAILED.value,
+    ):
+        sk_list = validation_item.sk.split("#")
+        assert len(sk_list) == 4, sk_list
+
+        errors.append(
+            {
+                "check_type": sk_list[1],
```

Comment:
Could we call this just "check" (same as elsewhere) or "check_name" (maybe more user friendly?)?